src/app/search/search.component.spec.ts

```diff
@@ -20,7 +20,8 @@ const CASE_TYPES: CaseType[] = [
         states: [],
         events: [],
         case_fields: [],
-        jurisdiction: null
+        jurisdiction: null,
+      printableDocumentUrl: 'print-url'
```

Comment:
If this refers to a set of documents and a single url, the name should be printableDocumentsUrl.
If this refers to a set of document urls, the name should be printableDocumentUrls.
if this refers to just one document url, please ignore this comment.

Reply:
This comment kindly applies to the other occurrences in the other files under this PR.

Reply:
Good spot!   I renamed the field to printableDocumentsUrl as per the field declared in definitions API in all places.